docs/source/user_guide/integrations/delta_lake.rst

```diff
@@ -0,0 +1,123 @@
++-----------------------------+------------------------------------------------------------------------------------------+
+| `float`                     | :meth:`daft.DataType.float32() <daft.DataType.float32>`                                  |
++-----------------------------+------------------------------------------------------------------------------------------+
+| `double`                    | :meth:`daft.DataType.float64() <daft.DataType.float64>`                                  |
++-----------------------------+------------------------------------------------------------------------------------------+
+| `decimal(precision, scale)` | :meth:`daft.DataType.decimal128(precision, scale) <daft.DataType.decimal128>`            |
++-----------------------------+------------------------------------------------------------------------------------------+
+| `date`                      | :meth:`daft.DataType.date() <daft.DataType.date>`                                        |
++-----------------------------+------------------------------------------------------------------------------------------+
+| `timestamp`                 | :meth:`daft.DataType.timestamp(timeunit="us", timezone=None) <daft.DataType.timestamp>`  |
++-----------------------------+------------------------------------------------------------------------------------------+
+| `timestampz`                | :meth:`daft.DataType.timestamp(timeunit="us", timezone="UTC") <daft.DataType.timestamp>` |
++-----------------------------+------------------------------------------------------------------------------------------+
+| `string`                    | :meth:`daft.DataType.string() <daft.DataType.string>`                                    |
++-----------------------------+------------------------------------------------------------------------------------------+
+| `binary`                    | :meth:`daft.DataType.binary() <daft.DataType.binary>`                                    |
++-----------------------------+------------------------------------------------------------------------------------------+
+| **Nested Types**                                                                                                       |
++-----------------------------+------------------------------------------------------------------------------------------+
+| `struct(fields)`            | :meth:`daft.DataType.struct(fields) <daft.DataType.struct>`                              |
++-----------------------------+------------------------------------------------------------------------------------------+
+| `list(child_type)`          | :meth:`daft.DataType.list(child_type) <daft.DataType.list>`                              |
++-----------------------------+------------------------------------------------------------------------------------------+
+| `map(K, V)`                 | :meth:`daft.DataType.struct({"key": K, "value": V}) <daft.DataType.struct>`              |
```

Comment:
We should update the map type to @colin-ho 's new dedicated Map type

Reply:
Good catch!